core/templates/pages/blog-dashboard-page/modal-templates/author-detail-editor-modal.component.spec.ts

```diff
@@ -114,6 +114,20 @@ describe('Upload Blog Post Thumbnail Modal Component', () => {
       'Author Bio should not be empty.'
     ]);
 
+    component.authorName = 'A';
+    component.authorBio = 'general bio';
+    expect(component.validateAuthorDetails().length).toBe(1);
+    expect(component.validateAuthorDetails()).toEqual([
+      'Author Name should not be less than 2 characters.',
+    ]);
+
+    component.authorName = 'test username';
+    component.authorBio = 'A';
+    expect(component.validateAuthorDetails().length).toBe(1);
+    expect(component.validateAuthorDetails()).toEqual([
+      'Author Bio should not be less than 5 characters.',
+    ]);
```

Comment:
Please add a test case where both `authorName` and `authorBio` are valid.

Reply:
I believe there is already a test to check for the case when `authorName `and `authorBio` are valid. Correct me if I am wrong. Thanks
https://github.com/oppia/oppia/blob/a47be7c50d40c52711e349549e9738e71cf66ee1/core/templates/pages/blog-dashboard-page/modal-templates/author-detail-editor-modal.component.spec.ts#L98-L100